app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,276 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "font-awesome";
+
+$teal: #159E98;
+$other-highlight: #218578;
+$teal-grey: #32A69B;
+$dark-teal: #238479;
+$rust: #C18763;
+$yellow: #D6D8A6;
+
+$light-contrast: $teal-grey;
+$flash: #75E8E1;
+$highlight: $teal;
+$neutral: $yellow;
+
+/*$body: 'IM Fell DW Pica';*/
+$body: 'Tinos';
+$header: 'Tinos';
+
+.each-merchant {
+  padding-bottom: 70px;
+}
+
+.merchant-name {
+  padding-bottom: 20px;
+  text-align: center;
+  max-width: 380px;
+}
+
+.centerBlock {
+  display: table;
+  margin: 0 auto;
+  margin-top: 20px;
+  margin-bottom: 20px;
+            }
+
+.product-names {
+  text-align: center;
+}
+
+.emptycart {
+  text-align: center;
+}
+
+.flash {
+  background-color: $flash;
+  text-align: center;
+  font-size: 16px;
+  padding: 5px;
+  margin: 5px;
+}
+
+.front-pg {
+  text-align: center;
+  font-family: $body, serif;
+  font-size: 64px;
+}
+
+.front-pg-link {
+  color: black;
+}
+
+.header-box {
+  padding: 20px;
+  background-color: $light-contrast;
+  margin-bottom: 10px;
+  margin-top: 10px;
+}
+
+.button-box {
+  padding-bottom: 25px;
+}
+
+.col-xs-12 {
+  text-align: center;
+}
+
+body {
+  font-family: $body, serif;
+}
+
+.front-pg2 {
+  font-family: $header, serif;
+  text-align: center;
+  font-size: 20px;
+}
+
+.jumbotron {
+  background-image: image-url("quartzy.png");
+  background-size: cover;
+
+}
+
+.home_img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto
+}
+
+.cntr {
+  text-align: center;
+}
+
+h1 {
+    font-family: $header, serif;
+    text-align: center;
+    margin-bottom: 20px;
+}
+
+h2 {
+    font-family: $header, serif;
+}
+
+h3 {
+    font-family: $header, serif;
+}
+
+h4 {
+    font-family: $header, serif;
+}
+
+h5 {
+    font-family: $header, serif;
+}
```

Comment:
Agreed. I know it's super hard to keep spacing and indentation consistent between four people and in such a short period of time. More of something to think about, as (at least I feel) there are much more important things to learn in the limited amount of time you guys have on a project. It may be easier when you get into your internships because some companies have specific style guides they follow. Consistent style throughout a project makes everything much easier to read.